web/src/components/majorRadio/majorRadio.js

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from 'react' 
+import './majorRadio.css'
+
+class MajorRadio extends Component {
+  
+  render() {
+    let cn = "majorRadio-wrapper"
+    
+    switch (this.props.valid) {
```

Comment:
switch could be if, see above

Reply:
Originally it was supposed to be more than a boolean, but since it doesn't bring big advantage or disadvantage in readability and performance I don't see the point in changing it.

Reply:
I disagree about the readability. A `switch` is less clear than `if` if the value to be compared is a boolean. Foremost, `switch` implies that there could be a default case where no other case was matched (you even added a `default`), even though that is not the case with a boolean. For the sake of code readability and maintainability I believe you should change this.